tools/util/injectorutil.go

```diff
@@ -557,3 +575,138 @@ func injectExecution(
 		log.L().Error("Failed to inject execution", zap.Error(err))
 	}
 }
+
+//GetAllBalanceMap returns a account balance map of all admins and delegates
+func GetAllBalanceMap(
+	client iotexapi.APIServiceClient,
+	admins []*AddressKey,
+	delegates []*AddressKey,
+) (map[string]*big.Int, error) {
+	balanceMap := make(map[string]*big.Int)
+	for _, admin := range admins {
+		addr := admin.EncodedAddr
+		err := backoff.Retry(func() error {
+			acctDetails, err := client.GetAccount(context.Background(), &iotexapi.GetAccountRequest{Address: addr})
+			if err != nil {
+				return err
+			}
+			balanceMap[addr] = big.NewInt(0)
+			balanceMap[addr].SetString(acctDetails.GetAccountMeta().Balance, 10)
+			return nil
+		}, backoff.NewExponentialBackOff())
+		if err != nil {
+			log.L().Fatal("Failed to Get account balance",
+				zap.Error(err),
+				zap.String("addr", admin.EncodedAddr))
+		}
+	}
+	for _, delegate := range delegates {
+		addr := delegate.EncodedAddr
+		err := backoff.Retry(func() error {
+			acctDetails, err := client.GetAccount(context.Background(), &iotexapi.GetAccountRequest{Address: addr})
+			if err != nil {
+				return err
+			}
+			balanceMap[addr] = big.NewInt(0)
+			balanceMap[addr].SetString(acctDetails.GetAccountMeta().Balance, 10)
+			return nil
+		}, backoff.NewExponentialBackOff())
+		if err != nil {
+			log.L().Fatal("Failed to Get account balance",
+				zap.Error(err),
+				zap.String("addr", delegate.EncodedAddr))
+		}
+	}
+
+	return balanceMap, nil
```

Comment:
there is no case an error will be returned, should we delete error or return error on the above two fatal cases?